content/applications/hr/payroll.rst

```diff
@@ -357,3 +371,27 @@ Resume
    Currently, the :guilabel:`Resume` feature found inside the :menuselection:`Payroll app -->
```

Comment:
I wouldn't lead with an admonition black, it needs a summary before.

Reply:
This was suggested by Zac since this is not a super functional section for most companies who use Odoo.  I believe this is going to change in the next big release, so this can be edited when this is expanded/changed in the next version. 

I do agree with you- typically, we would not have this, but I put this in since it was specifically told that I could do this as a work-around for this particular section.